cluster/gce/gci/configure-helper.sh

```diff
@@ -479,7 +479,11 @@ function ensure-local-ssds-ephemeral-storage() {
   systemctl stop ${container_runtime}
   # Some images mount the container runtime root dir.
   umount "/var/lib/${container_runtime}" || true
-  mkdir -p "${ephemeral_mountpoint}/${container_runtime}"
+  # Move the container runtime's directory to the new location to preserve
+  # preloaded images.
+  if [ ! -d "${ephemeral_mountpoint}/${container_runtime}" ]; then
+    mv "/var/lib/${container_runtime}" "${ephemeral_mountpoint}/${container_runtime}"
```

Comment:
We unmounted "/var/lib/${container_runtime}" above. Is there any data here to move?

Reply:
They are rw remounts. I have yet to verify in Ubuntu, though. I'll get back to this.

Reply:
Sorry, I don't follow. Do you mean that the cases where there are preloaded images, /var/lib/docker isn't mounted so the unmount is a no-op?

Reply:
I rechecked. The OS images that mount are the containerd ones. And it's just a RW remount in the same disk. We need to unmount prior to moving the folder.

This is independent of having preloaded images or not.

Reply:
I'm still not following. Why do we need to unmount? If we're concerned about someone writing to it during the move, the umount isn't enough because it will silently fail if the dir is already in use (ie there's a race).

Reply:
We need to unmount to be able to move the data. Otherwise we would get a "resource busy" error. Nothing is writing to it, as the container runtime is stopped.

Reply:
finally I understand, bind-mounted dirs make that resource busy error. thx